src/js/api/core-shim.js

```diff
@@ -94,6 +94,7 @@ Object.assign(CoreShim.prototype, {
             return new Providers(configuration);
         };
         model.setProvider = function() {};
+        setLiveSyncDurationAttribute(model);
```

Comment:
I thought this would be a good place for this because this is where the model is created and also where the other config properties are copied to the model